### ISSUE TITLE
Change the location of the various files to be compliant to the Filesystem Hierarchy Standard

### ISSUE THREAD

**Issue by SamuelLongchamps** [SamuelLongchamps/OpenBoard#13]
The current location where the package is installed (/usr/local/OpenBoard-%VERSION) has to be changed since it makes no sense (no other software is installed in that location). Some things are to be done to comply to the standard:
- Remove the version from the folder name and the .desktop file's "name" field
- Only put read-only files under /usr/*\* as per the Filesystem Hierarchy Standard
- Move the configuration file(s) to /etc
- Remove the need for the "run.sh" script which sets the LD_LIBRARY_PATH in order to move the executable to /usr/bin/ (since /usr/share/ should be architecture independent)

**Comment by watsaig** [comment]
Agreed. In general the packaging scripts need some serious re-writing. I think they've been that way since the beginnings of the project.
So far I only made minimal changes so that they would just work, but it would be nice to clean everything up.
One other problem I just noticed the other day is that if the Qt libraries are included in the package, they are still listed as dependencies, which sort of defeats the purpose.

**Comment by SamuelLongchamps** [comment]
I've started a rewrite, but as of now I've hardcoded the dependencies to get around the Qt libraries being listed as you said. I couldn't get the version for many packages in the original script and it would produce a package name followed by empty parentheses. I think using python as a script language could dramatically increase readibility and further customizations, like a rewrite of the dynamic library dependencies listing.

As for the location of files, this would probably take a bit more time since it affects more than the packaging process. I've added a symlink in /usr/bin/ for now to at least be able to easily launch the application from the terminal (and as a simple command for the .desktop file).

**Comment by watsaig** [comment]
Je me suis basé sur ce que tu as fait pour mettre un peu tout ça à jour (merci encore!).

Pour le moment, j'ai:
- séparé buildDebianPackage.sh en build.sh et package.sh (comme toi mais à la différence que les resources, librairies et plugins qt etc sont copiés par package.sh, donc build.sh ne s'occupe vraiment que de la compilation)
- intégré le symlink dans /usr/bin
- intégré [ton .xml](https://github.com/SamuelLongchamps/OpenBoard/issues/11#issuecomment-207927090) pour associer les fichiers ubz à OpenBoard, et ajouté les commandes nécessaires à `postinst` pour qu'OpenBoard ouvre les ubz par défaut
- corrigé la copie des librairies et plugins Qt, donc maintenant toutes les librairies et plugins nécessaires sont inclus dans le paquet et tout s'exécute correctement, en tout cas sur une config Ubuntu sans Qt5 (je n'ai pas encore testé les éventuels conflits dans le cas où une version antérieure de Qt est déjà installée)
- corrigé la génération des dépendances pour que les librairies Qt n'y figurent pas
- En passant, j'ai fait un peu le ménage et déplacé les scripts de déploiement dans un nouveau dossier

Et j'ai également changé le répertoire d'installation pour /opt/openboard/. Ce n'est pas idéal mais c'est toujours mieux qu'avant, et ça nous laisse le temps de faire les choses comme il faut d'ici la version 1.3.1 (ou suivante).

Btw, [Le commit en question](https://github.com/DIP-SEM/OpenBoard/commit/9520ddeb65753fb4022b8522ba506b2744886e08)

**Comment by SamuelLongchamps** [comment]
Entendu, je vais le tester aujourd'hui et je mergerai avec mon repo. Si je trouve des améliorations je ferai un pull request. Est-ce que l'ouverture des fichiers .ubz est fonctionnelle avec l'association des .ubz? Je n'avais pas réussi à le faire fonctionner (bug https://github.com/SamuelLongchamps/OpenBoard/issues/11).

**Comment by watsaig** [comment]
Oui, les ubz s'ouvrent bien avec OpenBoard -- par contre ils n'ont pas d'icône, je n'arrivais pas à faire ça (bon, j'ai abandonné après 5 minutes, ça doit quand même être faisable)
